.gitignore

```diff
@@ -53,3 +54,10 @@ google-services.json
 freeline.py
 freeline/
 freeline_project_description.json
+
+# MAC OS generated files
```

Comment:
I'd avoid putting these into a gitignore, the mac user should have them in his own global file.

Reply:
ok, i will change that.